hanadb_exporter/main.py

```diff
@@ -97,10 +98,12 @@ def run():
         )
     except KeyError as err:
         raise KeyError('Configuration file {} is malformed: {} not found'.format(args.config, err))
-
+    logger.info('connection to hanadb sucessfully initialized')
     collector = exporter_factory.SapHanaExporter.create(
         exporter_type='prometheus', metrics_file=metrics, hdb_connector=connector)
     REGISTRY.register(collector)
+    logger.info('exporter sucessfully registered')
+    logger.info('starting to serving metrics')
```

Comment:
This would be `starting to server metrics`, isn't it?

Reply:
:man_facepalming: